ribbon/src/test/java/feign/ribbon/LoadBalancingTargetTest.java

```diff
@@ -71,10 +71,43 @@ public void loadBalancingDefaultPolicyRoundRobin() throws IOException, Interrupt
       getConfigInstance().clearProperty(serverListKey);
     }
   }
+  
+  @Test
+  public void loadBalancingTargetPath() {
+    String name = "LoadBalancingTargetTest-loadBalancingDefaultPolicyRoundRobin";
+    String serverListKey = name + ".ribbon.listOfServers";
+  
+    server1.enqueue(new MockResponse().setBody("success!"));
+    server2.enqueue(new MockResponse().setBody("success!"));
+  
+    getConfigInstance().setProperty(serverListKey,
+            hostAndPort(server1.url("").url()) + "," + hostAndPort(
+                    server2.url("").url()));
+  
+    try {
+      LoadBalancingTarget<TestInterface>
+              target =
+              LoadBalancingTarget.create(TestInterface.class, "http://" + name + "/servers");
+      TestInterface api = Feign.builder().target(target);
+    
+      api.get();
+      api.get();
+    
+      assertEquals(1, server1.getRequestCount());
```

Comment:
I think you should do an assertion on the path received by the server. ex server1.takeRequest().getPath()